### DISCUSSION TITLE
Welcome to geoclimate Discussions!

### DISCUSSION THREAD

**Discussion by ebocher** [orbisgis/geoclimate#585] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 4estone** [comment]
Did you plan to upgrade geoclimate in order to process BD Topo V3?

**Reply by j3r3m1** [reply]
> Did you plan to upgrade geoclimate in order to process BD Topo V3?

Please open a new discussion in https://github.com/orbisgis/geoclimate/discussions
